blog/2023-07-12-angular-tutorial.md

```diff
@@ -0,0 +1,614 @@
+  </footer>
+
+  ...
+</div>
+
+...
+```
+
+## Step 8: Translation
+
+Once we have added the T components, we can begin translating any part of our website by simply hovering the text and performing `alt+click` or `option+click`.
+
+On `alt` text hover, the text to be translated will be highlighted like in the screenshot below
+
+![alt click](/img/blog/angular-tutorial/alt_click.png)
+
+If you click on the text while still holding the `alt` key, a dialog box will open for you to translate your text.
+
+![Dialog translation](/img/blog/angular-tutorial/angular_quick-trans.png)
+
+Translate your text and click on `create` to save the translation into the Tolgee Platform. Repeat this action for every text you want to translate.
+
+This approach is faster than to edit the localization `json` file or add it to the platform manually. To fix such key, you just have to click the string and any team member can do that with Tolgee chrome plugin, not only developers. You can [generate screenshots](https://tolgee.io/js-sdk#automated-screenshots-generation) by clicking the camera. This way future translators will know the context and will translate the string correctly
+
+Note that you can also manually add translations to your projects via the Tolgee Platform and export them.
+
+![Add translation button](/img/blog/angular-tutorial/addtrans_1.png)
+
+In the dialog that opens, add your translations and save
+
+![Create new key](/img/blog/angular-tutorial/addtrans_2.png)
+
+Make sure to select all the languages you want to add a translation for. In this case, we want translations for our three languages `en`, `fr`, and `es`. Add the key name of your translation in the `Key` field. In the screenshot above the key `welcome` corresponds to the word `Welcome` in English. Once you click on the `French` or `Spanish` field, it automatically suggests you a list of translations to choose from various services like AWS translate or Google Translate and you can easily customize it to fit your context.
+
+![Tranlate text](/img/blog/angular-tutorial/addtrans_3.png)
+
+Repeat the process for every piece of text you want to translate. At the end your translation screen should be close to this:
+
+![Translation list](/img/blog/angular-tutorial/translation_list.png)
+
+In case you have a lot of translations from large codebase, you can use the [CLI extract command](https://tolgee.io/tolgee-cli/extraction/syncing-strings). (It works only for React now, but we are working on Vue and Angular implementations)
+
+To export your translations, click on the Export menu in your left menu
+
+![Export Translation files](/img/blog/angular-tutorial/export_Trans.png)
+
+Select all the languages you wish to export and click on `export`. This will save a compressed file on your machine.
+
+Now, navigate to `src` folder in your angular project and create `i18n` folder. Add the content of the extracted file to this folder.
+
+![Create I18n folder](/img/blog/angular-tutorial/angular_i18n.png)
```

Comment:
You don't have to show the full contents of the files here. It's very long for the article.